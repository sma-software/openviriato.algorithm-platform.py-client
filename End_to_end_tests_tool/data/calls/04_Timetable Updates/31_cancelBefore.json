{
    "restResource": "/trains/{trainID}/train-path-nodes/{trainPathNodeID}:cancel-before",
    "httpMethod": "PUT",
    "urlParameters": [ "=cancelTrainTo:ID=", "=cancelTrainTo:TrainPathNodes[1].ID=" ],
    "requestBody": null,
    "CSharpMethodName": "CancelTrainBefore(trainID, trainPathNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64", "System.Int64" ],
    "subcategory": "Cancel Train Before"
}
