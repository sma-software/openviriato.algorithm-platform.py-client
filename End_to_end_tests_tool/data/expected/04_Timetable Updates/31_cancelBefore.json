{"ID":8120,"TrainPathNodes":[{"ID":8118,"SectionTrackID":null,"NodeID":15,"NodeTrackID":16,"FormationID":1187,"ArrivalTime":"2003-09-01T00:14:00","DepartureTime":"2003-09-01T00:14:00","MinimumRunTime":null,"MinimumStopTime":"P0D","StopStatus":"Passing","SequenceNumber":0},{"ID":8119,"SectionTrackID":1174,"NodeID":10,"NodeTrackID":12,"FormationID":null,"ArrivalTime":"2003-09-01T00:19:00","DepartureTime":"2003-09-01T00:19:00","MinimumRunTime":"PT5M","MinimumStopTime":"P0D","StopStatus":"CommercialStop","SequenceNumber":1}],"DebugString":"RVZH_13_1_J03 tt_(S)"}